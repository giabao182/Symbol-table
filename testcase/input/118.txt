INSERT a number
ASSIGN a 1
BEGIN
INSERT a string
ASSIGN a 'hello'
ASSIGN a 100
END
ASSIGN a 5