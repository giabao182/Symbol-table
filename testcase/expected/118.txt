success
success
success
success
TypeMismatch: ASSIGN a 100
success